labellab-flask/api/middleware/logs_decorator.py

```diff
@@ -160,6 +160,15 @@ def wrap(*args, **kwargs):
         issue = find_issue_by_id(issue_id)
         message = f'Issue {issue["title"]} assigned to {user["name"]}'
         category = 'issues'
+     
+      # Comment controller
+      if url.startswith('/api/v1/comment/create/'):
+        issue_id = kwargs.get('issue_id')
+        issue = find_issue_by_id(issue_id)
+        project_id = issue['project_id']
+        message = f'{user["username"]} posted a new comment on issue {issue_id}'
```

Comment:
Doesn't make much sense to show issue id in message. Show the project title instead of it

Reply:
Ah shit I meant issue title. Will fix this one myself.